packages/docs/source/building-blocks/elements.md

```diff
@@ -2,6 +2,43 @@
 
 Odyssey takes care to provide additional style to the following HTML elements by default.
 
+
+## abbr <a name="abbr"></a> 
+> The HTML `<abbr>` element represents an abbreviation or acronym; the optional title attribute can provide an expansion or description for the abbreviation. If present, title must contain this full description and nothing else. - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr'>MDN</a></cite>
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <p>You can use <abbr title="Cascading Style Sheets">CSS</abbr> to style your <abbr title="HyperText Markup Language">HTML</abbr>.</p>
+  </div>
+
+```html
+<p>You can use <abbr title="Cascading Style Sheets">CSS</abbr> to style your <abbr title="HyperText Markup Language">HTML</abbr>.</p>
+```
+</figure>
+
+## address <a name="address"></a> 
+> The HTML `<address>` element indicates that the enclosed HTML provides contact information for a person or people, or for an organization. - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address'>MDN</a></cite>
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <p>Contact the author of this page:</p>
+
+    <address>
```

Comment:
This is rendering weird for me:

<img width="482" alt="Screenshot 2020-04-06 10 11 42" src="https://user-images.githubusercontent.com/36284167/78585878-6e525500-77ef-11ea-98f1-d3520c256b26.png">

Reply:
Fixed this, had to do with spacing in markdown which affected the output